packages/cli/src/commands/generate/auth/providers/firebase.js

```diff
@@ -0,0 +1,30 @@
+export const config = {
+  imports: [
+    { import: '* as firebase', from: 'firebase/app' },
+    { import: null, from: 'firebase/auth' },
```

Comment:
It might be more readable to change this data structure, I don't particularly see a reason why we want to split these imports into something like we currently have:

```suggestion
    `import * as firebase 'firebase/app'`,
    `import 'firebase/auth'`,
```

This of course means that we need to modify the existing auth generators.

Reply:
@peterp I don't see either and I prefer your proposition, I'd see the original intent but finding a more elegant way might not be worth the final costs ( in implementation and use ).
Thanks a lot for your inputs btw :)

@cannikin what do you think about it? Should we change how imports are handled? If so I can send a commit fixing this before next week.

Reply:
Yep I'm down with changing it, looks like my original implementation wasn't as flexible as we'll need going forward.

Reply:
@cannikin ok, that part is fixed, I modified other providers accordingly.

@peterp we will need to update the documentation on adding a new provider, since @jtoar changes to CLI for each new provider we need to add it as a possible value for the generate command. I'd fix it but there's some ongoing work on the auth doc I don't have access to.

Reply:
@noire-munich It looks like you already added it as a possible value for the generate command:
```
choices: ['netlify', 'auth0', 'magic-link', 'firebase'],
```
do you mean update the [Contributing](https://github.com/redwoodjs/redwood/tree/master/packages/auth#contributing) section of the auth readme (which is the same doc that's on redwoodjs.com) with something like "make sure to add the new provider to the `choices` array"?

Reply:
> do you mean update the Contributing section of the auth readme (which is the same doc that's on redwoodjs.com) with something like "make sure to add the new provider to the choices array"?
yes @jtoar  that's what I meant :), and indeed, I had to put it into my PR otherwise no generation ( and manual test ) possible.